packages/flame_tiled/lib/src/renderable_tile_map.dart

```diff
@@ -103,9 +103,10 @@ class RenderableTiledMap {
     return result;
   }
 
-  void _fillBatches() {
+ /// Uptates the cache
+  void update() {
```

Comment:
This should have a more specific name to not be confused with a `Component.update` method

Reply:
OK now?